src/modules/blogs/blogs.service.ts

```diff
@@ -46,4 +47,28 @@ export class BlogService {
       created_at: savedBlog.created_at,
     };
   }
+
+  async getSingleBlog(blogId: string, user: User): Promise<any> {
```

Comment:
The promise should have a shape not just any

Reply:
Done @Cobidex